### ISSUE TITLE
Your New Logo

### ISSUE THREAD

**Issue by blindpet** [beville/ComicStreamer#27]
I am making guides for your software for use on Windows and the Raspberry Pi. Your logo was a fish and so I threw together something simple. I use square dimensions, feel free to use it http://www.mediafire.com/view/8k7t79rc4nkx78z/comicstreamer-logo-newer.png

**Comment by nomandera** [comment]
Typically you should define copyright and license or gift it to the project outright so they can own the copyright and choose the license.

Otherwise its a confusion waiting to happen.

Also you should link to the vector version as well so that it can be losslessly scaled.

Nice work

**Comment by blindpet** [comment]
Agreed @anoma I do have the vector version, I should have said it was a draft, still finalizing it and a vector will be provided when it's completely done.

It will be a gift as long as I can use it for guides :)

**Comment by blindpet** [comment]
Here is an updated version I had someone with a bit more talent do one. png and vector below, this is the one I will use for the Windows guide (git version so it is updateable)

http://imgur.com/jsEXwF6

http://www.mediafire.com/download/9rz203it2lbl17k/comicstreamer_with_text_2.svg
